Missing jinja2 and dask[diagnostics] packages when running lsdb.read_hipscat()

I am running a notebook in which I'm using lsdb 0.1 (installed with pip into a Python 3.11.7 conda environment). Upon using `lsdb.read_hipscat()` I get an error `ModuleNotFoundError: No module named 'jinja2'`.

Then the traceback later says

```
The above exception was the direct cause of the following exception:
```

followed by 

```
ImportError: Dask diagnostics requirements are not installed.

Please either conda or pip install as follows:

  conda install dask                     # either conda install
  python -m pip install "dask[diagnostics]" --upgrade  # or python -m pip install
```

Shouldn't a `pip install lsdb` have installed these for me?

In case it's related, running `pytest` on lsdb with only the runtime requirements triggers the missing `jinja2 error.

```
$PREFIX/lib/python3.11/site-packages/dask/widgets/__init__.py:4:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from __future__ import annotations

    import datetime
    import html
    import os.path

>   from jinja2 import Environment, FileSystemLoader, Template
E   ModuleNotFoundError: No module named 'jinja2'

$PREFIX/lib/python3.11/site-packages/dask/widgets/widgets.py:7: ModuleNotFoundError

The above exception was the direct cause of the following exception:

small_sky_order1_catalog = Dask DataFrame Structure:
                         id       ra      dec ra_error dec_error Norder    Dir   Npix
nparti...    ...      ...      ...      ...       ...    ...    ...    ...
Dask Name: read_parquet_file_to_pandas, 1 graph layer

    def test_catalog_html_repr_equals_ddf_html_repr(small_sky_order1_catalog):
>       assert small_sky_order1_catalog._repr_html_() == small_sky_order1_catalog._ddf._repr_html_()

tests/lsdb/catalog/test_catalog.py:23:
```
That import seems to be coming from `dask.widgets`, which makes me think it's a dask dependency bug (they forgot to require jinja2)?

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -16,8 +16,10 @@ classifiers = [
 ]
 dynamic = ["version"]
 dependencies = [
-    # dask distributed is included to ease creation of parallel dask clients.
-    "dask[dataframe,distributed]",
+    # Includes dask[array,dataframe,distributed,diagnostics].
+    # dask distributed eases the creation of parallel dask clients.
+    # dask diagnostics is required to spin up the dashboard for profiling.
+    "dask[complete]",
     "hipscat",
     "pyarrow",
     "deprecated",

